Formula/fasthenry-wr.rb

```diff
@@ -0,0 +1,34 @@
+class FasthenryWr < Formula
```

Comment:
Is this the same software as https://repology.org/project/fasthenry/versions?

Reply:
Yes and no. They are all `fasthenry`, but even on that repology site it looks like they all might be different forks of `fasthenry`.